base/intset.jl

```diff
@@ -220,6 +220,12 @@ function ==(s1::IntSet, s2::IntSet)
     return true
 end
 
+"""
+    issubset(A, S) -> Bool
+    ⊆(A,S) -> Bool
+
+Return `true` if `A` is a subset of or equal to `S`.
```

Comment:
The docstring for `issubset` in set.jl specifies the use of `in`; `in` is used in the set.jl definition but not the intset.jl definition.

If that distinction is not important, I can delete the specification and also remove the docstring from intset.jl. What do you think?